docker/multi-process/README.md

```diff
@@ -23,7 +23,9 @@ and
 Its recommended to use an image that allows you to create a database via environmental variables at docker run, like `paintedfox / postgresql` or `centurylink / mysql`, so the db is populated when this script runs.
 
 If you do not link a database container, a built-in mysql database will be started.
-There is an exported docker volume of /var/lib/mysql to allow persistence of that mysql database.
+There is an exported docker volume of `/var/lib/mysql` to allow persistence of that mysql database.
+
+__NOTE:__ If you do not export the volme or use a linked database container you can not update Huginn without losing your data.
```

Comment:
"If you do not export the volume, or use a linked database container, you cannot update Huginn without losing your data."